### ISSUE TITLE
[FE] Send the parameter checkInType=preCheckIn when initiating the session for Pre Check-in scenario

### ISSUE THREAD

**Issue by randomsync** [department-of-veterans-affairs/va.gov-team#35834]
## How to configure this issue

- [ ] Attached to an Epic (what body of work is this a part of?)
- [ ] Correctly Labeled
- [ ] Fill in User Stories
- [ ] Fill in Documentation
- [ ] Fill in Tasks
- [ ] Fill in Acceptance Criteria

---

## User Story
To configure DataDog dashboard to show when a pre check-in is started, we need to send the parameter checkInType=preCheckIn so vets-api can differentiate between the calls made during pre check-in and day of check-in. We currently are showing all calls to session controller in vets-api (get and post) as day of check-in metrics.

This is the same as the parameter that is already being passed when pre check-in controller is called, which retrieves the data from LoROTA: https://github.com/department-of-veterans-affairs/vets-website/blob/32fddcb59f5741f3d5f36936d411b2a3a38daef3/src/applications/check-in/api/versions/v2.js#L85

> these should be why the veteran cares, and the veteran gets out of it.

- (%FILL IN%)
- (%FILL IN%)

## Related Documentation

> these should be links to feature docs

- [Sketch](https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e)
- [Product Docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin)
- [Feature Doc](%FILL IN%)

## Description

> this should be an understandable description of the ticket. This description should have enough context meaning that non-technical team members should get the idea.

Would be better to do this after this ticket, to just pass in the value in the store: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/36063

## Tasks

> this is a list of technical tasks that need for this ticket

- [ ] %Add any tasks%
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations
  - [ ] README
  - [ ] Use Cases

## Acceptance Criteria

> when is the ticket done

- [ ] %Add any additional Acceptance Criteria%
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by randomsync** [comment]
cc: @brianseek for visibility

**Comment by mdewey** [comment]
@randomsync This should be in staging, and deployed to production today, how can we verify this?

**Comment by randomsync** [comment]
@mdewey I was able to confirm that this parameter is being passed to GET /session. I was able to get the metric in DataDog (see "Pre Check In Started Count" [here](https://app.datadoghq.com/dashboard/tmn-f5f-e9r/check-in-vets-api)). However, I don't see it for POST /session. I see that you've added it [here](https://github.com/department-of-veterans-affairs/vets-website/pull/20130/files#diff-833459979b13ebba1e56c2bdca07024a3178512dedef1ac29fcd0927285429feR18), so it's probably something that needs to be updated in vets-api code.
